Resources/Prototypes/_LostParadise/Entities/Structures/Machines/vending_machines.yml

```
@@ -290,3 +290,36 @@
+      shader: unshaded
+    - state: panel
+      map: ["enum.WiresVisualLayers.MaintenancePanel"]
+  - type: PointLight
+    radius: 1.5
+    energy: 1.6
+    color: "#e0f1ff"
+  - type: Transform
+    noRot: false
+  - type: Rotatable
+    rotateWhileAnchored: false
+    rotateWhilePulling: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)